api-docs/graphql-api-elasticsearch

```diff
@@ -0,0 +1,13 @@
+---
+title: Search & Filter with the Trackunit Search Engine
+category: 652e3a8e279c3f001f9acdc3
+---
+
+In the Trackunit Iris platform we have integrated the Trackunit Search Engine powered by Elasticsearch with our GraphQL API. 
+This provides a powerful way for you to enhance your query capabilities. 
+By utilizing Elasticsearch as a data source within the GraphQL ecosystem, you can efficiently filter and retrieve the exact data from your fleet that you need. 
+
+When you initiate a GraphQL query, you can leverage Elasticsearch's robust querying capabilities, including full-text search, filtering, sorting, and aggregation. 
+By structuring your GraphQL queries with relevant Elasticsearch filters, such as range queries, term queries, or boolean queries, you can precisely narrow down the data to be retrieved.
+
```

Comment:
@mmt-trackunit I would really like to add an image here, but I don't know how to do this via github 😂  Would you mind adding the picture that is in this paper doc? https://www.dropbox.com/scl/fi/e6tipd1vinuzcmj0pc2wk/Product-Launch-Brief-PUBLIC-GraphQL-API.paper?rlkey=qvzq4g5156cwdedxysosn3fll&dl=0#:h2=Search-&-Filter-with-the-Track

Reply:
Added it now